lua/entities/gmod_wire_expression2/core/strfunc.lua

```diff
@@ -43,34 +40,36 @@ local function findFunc( self, funcname, typeids, typeids_str )
 		if not func then
 			for i=#typeids,1,-1 do
 				func, func_return_type = checkFuncName( self, funcname .. "(" .. concat(typeids,"",1,i) .. "...)" )
```

Comment:
It'd be nice if you could also replace these string concat messes with localized string.format calls

Reply:
I don't really know if
```lua
string.format("%s(%s:%s)", funcname, typeids[1], concat(typeids, "", 2))
```
is much better

Reply:
Not going for readability going for performance

Reply:
5 is a relatively small amount, not to mention the short length of the strings themselves
https://stackoverflow.com/questions/19138974/does-lua-optimize-the-operator